### PR TITLE
Add blacklist pr feature and refactor specs

### PR DESCRIPTION
**Pull request by vitogit**
The idea is to not send notifications from some people. So as we are not using the user table to log in users I will repurpose (create migration to delete unused fields and add new ones)  to blacklist users (not send notifications from this users PRs). 

We have the active admin running so we can access and create an user with github_name and blacklisted = true.

Also I refactored the specs

### CODE REVIEW

File: spec/requests/api/v1/github_webhook/filter_spec.rb [hunk 1]
```diff
@@ -200,3 +180,15 @@
     end
   end
 end
+
+def expect_notification(text: message)
```
Comment (dani24gl) [inline spec/requests/api/v1/github_webhook/filter_spec.rb]: Just curious, what is the colon meanning in a function parameter => `text: meesage`, Which is the reason why you don't use just the variable `message`?

Comment (vitogit) [reply spec/requests/api/v1/github_webhook/filter_spec.rb]: you can pass parameters in the form of expect_notification(text: 'something'), so it's more clear what parameter are you sending to the method, in this case :text

text: message means that the default value is message, in this case is a variable.

Comment (dani24gl) [reply spec/requests/api/v1/github_webhook/filter_spec.rb]: ok, I thought that the default values where defined like this: `def expect_notification(text = message)`

Comment (vitogit) [reply spec/requests/api/v1/github_webhook/filter_spec.rb]: It's a ruby 2.0 thing. Keyword arguments https://thoughtbot.com/blog/ruby-2-keyword-arguments

File: app/services/pull_request.rb [hunk 2]
```diff
@@ -34,4 +34,8 @@ def on_hold_label?(label)
     label.downcase == ON_HOLD.downcase
   end
 
+  def blacklisted?
+    User.where(github_name: username, blacklisted: true).present?
```
Comment (matiasmansilla1989) [inline app/services/pull_request.rb]: I think you can use https://apidock.com/rails/ActiveRecord/Base/exists%3F/class, not sure if the current solution brings the users to memory but I'm sure `exists` doesn't and it's more straightforward.

Comment (vitogit) [reply app/services/pull_request.rb]: Yes, that's better.

**Pull request merged**